frontend/src/components/panels/actionitems/WeeklyActionItemsPanel.tsx

```diff
@@ -0,0 +1,51 @@
+import { Paper, Typography } from '@mui/material';
+import ActionItemsTable, { ActionItemsColumn } from './ActionItemsTable';
+import { IActionItem } from 'shared-types';
+import ActionItemCheckbox from './ActionItemCheckbox';
+import './WeeklyActionItemsPanel.css';
+
+interface ActionItemsPanelProps {
+  data: IActionItem[];
+  height: number;
+  itemsID: string;
+  updateActionItemStatus: (item: IActionItem, itemsID: string) => void;
+}
+
+const WeeklyActionItemsPanel: React.FC<ActionItemsPanelProps> = ({ data, height, itemsID, updateActionItemStatus }) => {
```

Comment:
Add skeleton loading like other panels

Reply:
You can leverage the tanstack query `isLoading` which is true only for the first load (which is the only time skeleton loader is required)